src/lib/theme/variants/typography.ts

```diff
@@ -1,199 +1,31 @@
-import { fontSizes, lineHeights } from '../design-system';
-
-const displayLarge = {
-  lineHeight: lineHeights['6xl'],
-  fontSize: fontSizes['8xl'],
-  fontStyle: 'normal',
-  fontWeight: '400',
-  fontFamily: 'Proto',
-  '@media (min-width: 36em)': {
-    fontSize: fontSizes['9xl'],
-    lineHeight: lineHeights['6xl'],
-  },
-};
-
-const displayMedium = {
-  lineHeight: lineHeights['5xl'],
-  fontSize: fontSizes['7xl'],
+const display = {
   fontStyle: 'normal',
   fontWeight: '500',
   fontFamily: 'Proto',
-  '@media (min-width: 36em)': {
-    fontSize: fontSizes['8xl'],
-    lineHeight: lineHeights['6xl'],
-  },
 };
 
-const displaySmall = {
-  lineHeight: lineHeights['4xl'],
-  fontSize: fontSizes['6xl'],
-  fontStyle: 'normal',
-  fontWeight: '500',
+const headline = {
   fontFamily: 'Proto',
-  '@media (min-width: 36em)': {
-    fontSize: fontSizes['7xl'],
-    lineHeight: lineHeights['6xl'],
-  },
-};
-
-const headlineLarge = {
-  fontFamily: 'Proto',
-  fontSize: fontSizes['5xl'],
-  lineHeight: lineHeights['3xl'],
-  fontWeight: '500',
-  fontStyle: 'normal',
-  '@media (min-width: 36em)': {
-    fontSize: fontSizes['6xl'],
-    lineHeight: lineHeights['5xl'],
-  },
-};
-
-const headlineMedium = {
-  fontFamily: 'Proto',
-  fontSize: fontSizes['4xl'],
-  lineHeight: lineHeights['2xl'],
-  fontWeight: '500',
-  fontStyle: 'normal',
-  '@media (min-width: 36em)': {
-    fontSize: fontSizes['5xl'],
-    lineHeight: lineHeights['4xl'],
-  },
-};
-
-const headlineSmall = {
-  fontFamily: 'Proto',
-  fontSize: fontSizes['3xl'],
-  lineHeight: lineHeights['xl'],
-  fontWeight: '500',
-  fontStyle: 'normal',
-  '@media (min-width: 36em)': {
-    fontSize: fontSizes['4xl'],
-    lineHeight: lineHeights['3xl'],
-  },
-};
-
-const titleLarge = {
-  fontFamily: 'Satoshi',
-  fontSize: fontSizes['2xl'],
-  lineHeight: lineHeights['l'],
   fontWeight: '500',
   fontStyle: 'normal',
-  '@media (min-width: 36em)': {
-    fontSize: fontSizes['4xl'],
-    lineHeight: lineHeights['2xl'],
-  },
 };
 
-const titleMedium = {
+const title = {
   fontFamily: 'Satoshi',
-  fontSize: fontSizes['m'],
-  lineHeight: lineHeights['m'],
-  fontWeight: '700',
-  fontStyle: 'normal',
-  '@media (min-width: 36em)': {
-    fontSize: fontSizes['xl'],
-    lineHeight: lineHeights['xl'],
-  },
-};
-
-const titleSmall = {
-  fontFamily: 'Satoshi',
-  fontSize: fontSizes['s'],
-  lineHeight: lineHeights['s'],
   fontWeight: '500',
```

Comment:
On figma we had a twice Value to fontWeight: 500 and 700, its was a mistake or u decided to leave default 500